cegs_portal/static/css/project.css.tw

```diff
@@ -66,32 +66,61 @@
 
   a:visited {
     color: inherit;
+    text-decoration: none;
   }
 
   .th-bg-light-gray {
     background-color: rgb(249, 250, 251);
   }
 
-  table.data-table {
-    border-collapse: collapse;
-    width: 100%;
-    font-family: Helvetica, Roboto, sans-serif;
+  .table-title,
+  .svg-icon {
+      margin-right: 10px;
+      fill: currentColor;
+      height: 1em;
   }
 
-  table.data-table th {
-    padding: 12px;
-    text-align: left;
-    color: gray;
-    font-weight: bold;
+  .tab-bar {
+      background: #ffffff;
+      border-radius: 10px;
+      padding-left: 10px !important;
+      padding-right: 10px;
+      box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
   }
 
-  table.data-table td {
-    padding: 12px;
-    border-bottom: 1px solid lightgrey;
+  .container {
+    background: #ffffff;
+    box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
+    padding: 20px 40px;
+    margin-bottom: 20px;
+    border-radius: 10px;
+    width: 1100px;
```

Comment:
This property helps contain the volume of content in the container. If removed, the containers will lack uniformity because its size depends on the content volume. There are many ways to contain the content but this might be the quick/easier fix.

Reply:
Allowing the containers to change sizes based on content makes us lose the consistency and uniformity within the page. Consistency and uniformity in this case is pleasing to the eye. The size and consistency that everyone has been viewing seems to have gotten their approval through demos and presentations but we can also ask if they want a change it up. Maybe we should keep it as is or find a better way to get the same result.